query_optimizer/tests/execution_generator/Select.test

```diff
@@ -953,19 +953,79 @@ WHERE double_col < 0
 ==
 
 # Window Aggregation Test.
-# Currently this is not supported, an empty table will be returned.
-SELECT avg(int_col) OVER w FROM test
+SELECT char_col, long_col, avg(long_col) OVER w FROM test
 WINDOW w AS
-(PARTITION BY char_col
- ORDER BY long_col DESC NULLS LAST
+(ORDER BY char_col DESC NULLS LAST
```

Comment:
@zuyu I found that the code logic here is a little bit wrong. For window w/ NULLs, the NULL values should be ignored rather than directly mark the result NULL. Only when all the values in the window are NULLs should the window aggregate result be marked NULL. 

Will fix this in the next commit.